libs/isograph-react/src/loadable-hooks/useConnectionSpecPagination.ts

```diff
@@ -179,10 +195,7 @@ export function useConnectionSpecPagination<
 
   const getFetchMore =
     (after: string | null | undefined) =>
-    (
-      args: OmitFirstAfter<TReadFromStore['parameters']>,
-      first: number,
-    ): void => {
+    (first: number): void => {
       // @ts-expect-error
       const loadedField = loadableField({
         ...args,
```

Comment:
Since `fetchMore` is only called once (in response to a click, for example), and not on every render, I'm not super concerned about perf. But, I do think we can probably optimize this by avoiding spreading args unless this is the first call to `fetchMore` (i.e. if `state.length === 0`)

(Feel free to do that as a follow up, as well.)